src/query/service/src/servers/http/clickhouse_handler.rs

```diff
@@ -112,7 +112,14 @@ async fn execute(
     handle: Option<JoinHandle<()>>,
 ) -> Result<WithContentType<Body>> {
     let format_typ = format.typ.clone();
-    let mut data_stream = interpreter.execute(ctx.clone()).await?;
+    let mut data_stream = ctx
```

Comment:
It would be better to add some comments here to explain why we need run it in context runtime

Reply:
ok

Reply:
PTAL, the reason for spawning is documented, but rather lengthy,  failed to put it in a  concise way :(